nextflow.config

```diff
@@ -10,102 +10,98 @@
 params {
 
     // Workflow flags
-    help                       = false
-    input                      = "sample_sheet.tsv"
-    outdir                     = './results'
-    fasta                      = "data/*.fasta"
-    publish_dir_mode           = 'copy'
+    help                            = false
+    input                           = "sample_sheet.tsv"
+    outdir                          = './results'
+    fasta                           = "data/*.fasta"
 
     // References
-    genomes                    = null
+    genomes                         = null
 
     // Workflow options
-    allele_sheet               = false
-    include_proteins_from_vcf  = false
-    predict_class_1            = false
-    predict_class_2            = false
+    allele_sheet                    = null
+    include_proteins_from_vcf       = false
+    predict_class_1                 = false
+    predict_class_2                 = false
     refine_fdr_on_predicted_subset  = false
-    schema_ignore_params       = 'genome,input_paths'
-    skip_decoy_generation      = false
-    subset_affinity_threshold  = 500
-    variant_annotation_style   = "SNPEFF"
-    variant_frameshift_filter  = false
-    variant_indel_filter       = false
-    variant_reference          = "GRCH38"
-    variant_snp_filter         = false
+    schema_ignore_params            = 'genome'
```

Comment:
There is a duplicate in line 89:
`
    schema_ignore_params            = 'genomes,modules'`